script/metrics.js

```diff
@@ -49,6 +49,11 @@ function makeScoringGuide(curves) {
 }
 
 export const scoringGuides = {
+  // v7 scoring is identical to v6
+  v7: {
```

Comment:
given the version collapsing, this bit isn't necessary, but it feels good for anyone else consuming this code.